### ISSUE TITLE
Configurable location for custom dashboard assets

### ISSUE THREAD

**Issue by torkildr** [home-assistant/appdaemon#119]
In the process of creating a new widget, I'm starting to see the need of having custom assets in my conf-folder.

I see that custom css is probably already covered, but it would be really nice to have custom images and javascript as well.

For images, this could probably be solved by just adding a new add_static in appdash.py pointing to a configurable location.

For javascript assets, I'm thinking you'd also like a head tag to be generated.

I'd be happy to implement this, but think maybe we need to figure out what direction we should take first.

**Comment by acockburn** [comment]
Yes, good idea, I'll have a think about it for the next release.

**Comment by cgarwood** [comment]
Agreed, this would be helpful to have native support for. Right now I'm grabbing extra JS files from a local nginx webserver since I can't get it directly from hadashboard.

**Comment by torkildr** [comment]
I might use a few calories to implement this in a good way, seeing as there is interest for it.
